### PR TITLE
Mayank

### PR DESCRIPTION
**Pull request by MayankYadav11**
updated search page + movie page by accident.

### CODE REVIEW

File: data/tvshows.js [hunk 1]
```diff
@@ -1,58 +1,116 @@
...
 }
 
-async function searchTVShows(searchTerm) {
+async function searchMovie(searchTerm) {
 
     validation.validateString("searchTerm", searchTerm);
 
-    const data = await searchTVShowsAPI(searchTerm);
+    const data = await searchMovieAPI(searchTerm);
+    console.log("option a" + data.Search[0].imdbID);
```
Comment (n7rider) [inline data/tvshows.js]: Remove loggers, commented lines?

Comment (MayankYadav11) [reply data/tvshows.js]: yeah because we don't need searchtvshows or gettvshows anymore

File: data/tvshows.js [hunk 2]
```diff
@@ -1,58 +1,116 @@
...
 
-    validation.validateNumber("showID", showID);
+    validation.validateString("imdbID", imdbID);
 
-    const data = await getTVShowAPI(showID);
+    const data = await getMovieAPI(imdbID);
 
-    if (data.id == null) {
-        throw "TV Show not found for ID = `" + showID + "`";
+    if (data.Title == null) {
```
Comment (n7rider) [inline data/tvshows.js]: if(data.Title) should be enough

File: routes/movies.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+        res.render('shows/single', {
+            movie: movie,
+            title: movie.Title
+        });
+
+    } catch (e) {
+        if (e.response && e.response.status && e.response.status === 404) {
+            res.status(404).render('shows/error', {
+                title: "No TV Show Found",
```
Comment (n7rider) [inline routes/movies.js]: Replace TV show with movie? (Update everywhere in the file)

**Review by n7rider — APPROVED** [review]

File: routes/movies.js (hunk 3 above)
Comment (MayankYadav11) [reply routes/movies.js]: yeah forgot to do that. ill do that

**Pull request merged**